viewflow_extensions/management/commands/create_diagraph.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import, unicode_literals
+
+import importlib
+
+from django.conf import settings
+from django.core.management import BaseCommand, CommandError
+from six import text_type
+
+from ...flow_graph import FlowGraph
+
+
+class Command(BaseCommand):
+    help = 'Creates diagraph for the given flow.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('app', nargs=1, type=str,)
+        parser.add_argument('flow', nargs=1, type=str)
+        parser.add_argument("-s", "--svg", nargs='?',
+                            help="create dot file")
+
+    def handle(self, **options):
+        if not settings.DEBUG:
```

Comment:
Please remove the settings handling. I don't see a valid use case why this wouldn't work with `DEBUG=True`